Set up landing page view

callea2 opened the pull request:
PT Story: https://www.pivotaltracker.com/story/show/121650573
- Adds a landing page view with required necessities

callea2 committed:
Added barebone view
Generated controller, spec file and view
Set root path in routes

callea2 committed:
Modified index view

callea2 committed:
Merge branch 'development' of https://github.com/CraftAcademy/slow_food_online into feat_landing_page

callea2 committed:
Updated landing page
Added css
Added js
Converted index to haml

callea2 committed:
Minor index view update and added test coverage

tochman commented:
Couple of comments
- The categories and restaurants are in the database, so we should get them from there rather then hardcoding those values.
- We should have a path for what happens when there are no restaurants present
- In tests the Categories and Restaurants should be added to the test database in a Background step

Also, this code does not have the latest changes from `development`

callea2 committed:
Merged latest pull from upstream

callea2 commented:
Good ideas, I will start implementing the suggestions

callea2 committed:
Updated index view and landing controller

tochman committed:
refactores landing view markup
adds helper method to populate select field
refactores js code for landing_page
adds background to feature file
adds poltergaist ond phanom_js

callea2 committed:
WIP: Updated landing page view
Updated controller
Set new route
Updated cucumber tests

tochman committed:
fixes redirect to index and display of chosen categories
restaurants (name only)

callea2 committed:
Updated landing page tests

callea2 commented:
The implementations are now done thanks to Thomas, and the page now holds true to the story.

There's some minor stuff that should be adjusted in the future, but it's more about polishing.
